
-------
string job
string connection_status #To show connection status